index.js

```diff
@@ -25,6 +25,19 @@ export const config = {
 const usePreset = () => atom.config.get('linter-stylelint.usePreset');
 const presetConfig = () => atom.config.get('linter-stylelint.presetConfig');
 
+function getRange(data) {
```

Comment:
Btw, this function automatically handles when line and/or column are invalid (only throwing an error if the line or column are larger than the maximum).